build.xml

```diff
@@ -1558,8 +1565,8 @@ xmlns:cs="antlib:com.puppycrawl.tools.checkstyle.ant">
                classpathref="releaseaudit-classpath"/>
     </target>
 
-    <target name="owasp-taskdef" depends="ivy-retrieve-releaseaudit">
```

Comment:
Notice that the release audit jars are being dumped into build/lib - I'd recommend we fix that as part of this patch as well.

Reply:
+1